canonical-web-and-design/canonical.com
# Add search bar and search page

## Done

- Update vanilla dependency to 3.0 
- Add search bar on top navigation 
- Create `/search` page
- Add `canonicalwebteam.search` and update `app.py`


## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Please check the search bar works properly on the top nav (open, close, and overlay) 
- Please check the search result returns in the `/search` page. 

## Issue / Card

Fixes [#4894](https://github.com/canonical-web-and-design/web-squad/issues/4894) and [#4895](https://github.com/canonical-web-and-design/web-squad/issues/4895)
## Screenshots

<img width="1421" alt="image" src="https://user-images.githubusercontent.com/57550290/158607062-fd573a9b-6f8a-44df-8c0a-7926faad7662.png">
<img width="1424" alt="image" src="https://user-images.githubusercontent.com/57550290/158607103-e74f6dc9-247b-47df-b6ae-09afa1a82b28.png">
<img width="722" alt="image" src="https://user-images.githubusercontent.com/57550290/158607303-43243537-268c-4676-a612-ab2c7d47c482.png">
